core/src/test/java/org/apache/iceberg/TestRewriteManifests.java

```diff
@@ -151,7 +232,7 @@ public void testReplaceManifestsWithFilter() throws IOException {
     // get the file order correct
     List<DataFile> files;
     List<Long> ids;
-    try (ManifestReader reader = ManifestReader.read(localInput(manifests.get(0).path()))) {
+    try (ManifestReader reader = ManifestReader.read(manifests.get(0), table.io(), null)) {
```

Comment:
Maybe, we can overload this method to avoid passing null as a spec map.

Reply:
+1

Reply:
Done